aiohttp/web_fileresponse.py

```diff
@@ -37,6 +40,14 @@
 
 NOSENDFILE: Final[bool] = bool(os.environ.get("AIOHTTP_NOSENDFILE"))
 
+if sys.version_info < (3, 9):
+    mimetypes.encodings_map[".br"] = "br"
+
+# File extension to IANA encodings map that will be checked in the order defined.
+ENCODING_EXTENSIONS: Final[Dict[str, str]] = {
+    ext: mimetypes.encodings_map[ext] for ext in (".br", ".gz")
+}
```

Comment:
We try to avoid mutable globals:
```suggestion
ENCODING_EXTENSIONS: Final[MappingProxyType[str, str]] = MappingProxyType({
    ext: mimetypes.encodings_map[ext] for ext in (".br", ".gz")
})
```